lib/carousel.js

```diff
@@ -797,7 +799,7 @@ var Carousel = (0, _createReactClass2.default)({
       }
 
       if (index <= slidesAfter - 1) {
-        return (this.state.slideWidth + this.props.cellSpacing) * (this.state.slideCount + index);
+        return slidesBefore < 0 ? (this.state.slideWidth + this.props.cellSpacing) * (this.state.slideCount + index) : 0;
```

Comment:
This line did not work. I fixed to the below code, which worked:
`return slidesBefore < 0 ? (this.state.slideWidth + this.props.cellSpacing) * (this.state.slideCount + index) : (this.state.slideWidth + this.props.cellSpacing) * index;`

Reply:
return (this.state.slideWidth + this.props.cellSpacing) * (slidesBefore < 0 ? this.state.slideCount + index : index);

I was still seeing issues with stacked images at index 0. Updating line 800 to whats above appeared to have resolved our issues. If slides before is less than zero we want to use index * slide with. otherwise we want to use slideCount + index * slideWidth

Reply:
@kenwheeler - who do we need bribe to get this fixed. @ddcrogerk 's solution resolved the issue for me.

Reply:
@ddcrogerk Hmm looks like your code is doing the exact same thing as mine but shorter and cleaner :)